14-wt-machine-learning.Rmd

```diff
@@ -440,4 +464,8 @@ defaultSummary(as.data.frame(data_train_randomfor))
 ```
 Our output will come in the order we wrote the code, so the linear model output shows up above the random forest output.
 
-We can see that the random forest technique seems to perform better than regression. Specifically, the RMSE is lower for the random forest (4.80 as compared to 13.50 for the linear model). Second, the variance explained (`Rsquared`) is much higher in the random forest (0.96 as compared to 0.59 for the linear model). It may be interesting to compare the results from the random forest not to a more straightforward model, such as a regression, but to a more sophisticated model, like one for deep learning. As you expand your skills, you might be curious to do something like that!
\ No newline at end of file
+We can see that the random forest technique seems to perform better than regression. Specifically, the RMSE is lower for the random forest (4.80 as compared to 13.50 for the linear model). Second, the variance explained (`Rsquared`) is much higher in the random forest (0.96 as compared to 0.59 for the linear model). It may be interesting to compare the results from the random forest not to a more straightforward model, such as a regression, but to a more sophisticated model, like one for deep learning. As you expand your skills, you might be curious to do something like that.
+
+## Conclusion
+
+In this chapter, we introduced both general machine learning ideas, like a training and test data set and evaluating the importance of specific variables, and specific ideas, like how a random forest works and how to tune specific parameters so that the model is as effective as possible at predicting an outcome. Like many of the topics in this book - but, perhaps *particularly* so for machine learning - there is much more to learn on the topic, and we encourage you to consult the books and resources in the [learning more chapter](#c17) to learn about further applications of machine learning methods
```

Comment:
Love this conclusion, it really ties it all together. Well done. Maybe missing a period at the end!

Reply:
Thank you! Added.